client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/index.js

```diff
@@ -19,15 +20,18 @@ import {
 	isCustomsFormStepSubmitted,
 } from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
 import StepConfirmationButton from '../step-confirmation-button';
-import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+import getPackageDescriptions from '../packages-step/get-package-descriptions';
+import { getAllPackageDefinitions } from 'woocommerce/woocommerce-services/state/packages/selectors';
 
 const CustomsStep = ( props ) => {
 	const {
+		siteId,
+		orderId,
 		errors,
 		expanded,
 		translate,
 		isSubmitted,
-		dummyFieldChecked,
+		packageDescriptions,
 	} = props;
 	const summary = hasNonEmptyLeaves( errors ) ? translate( 'Customs information incomplete' ) : translate( 'Customs information valid' );
```

Comment:
This could be split into multiple lines:

```
const summary = hasNonEmptyLeaves( errors )
  ? translate( 'Customs information incomplete' )
  : translate( 'Customs information valid' );
```

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/fdb6090cac9e1605e7983dca29edeba971fea658